recipes/jove/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "jove" %}
+{% set version = "4.17.3.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jonmacs/jove/releases/download/{{ version }}/jove-{{ version }}.tgz
+  sha256: 94300bed2b73b3b964fe9e42c652175525609d3ebf6fdefd2291d277b5ae27a2
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: 'make -f Makefile.msc && cp jjove.exe "$PREFIX/bin/jove.exe"'  # [win]
+  script: 'make JOVEHOME="$PREFIX" SYSDEFS=-DDarwin install'  # [osx]
+  script: 'make JOVEHOME="$PREFIX" LDFLAGS="$LDFLAGS " TERMCAPLIB=-lncurses install'  # [linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - ncurses  # [not win]
+
+test:
+  commands:
+    - which jove
```

Comment:
Is there something like `jove --version` or `jove --help` that we could use to actually test that the binary has no missing dependencies somehow?

Reply:
Not that I could discover, sadly no. And once started, it seems to need user interaction to quit.